Cargo.toml

```diff
@@ -19,9 +19,9 @@ appveyor = { repository = "rust-lang-libs/regex" }
 
 [dependencies]
 # For very fast prefix literal matching.
-aho-corasick = "0.6.0"
+aho-corasick = { version = "0.6.0", default-features = false }
```

Comment:
I think this is blocked on aho-corasick getting a new release. `regex` would then need to depend on at least the newly released version.